build/amis/ansible/roles/containerd/defaults/main.yml

```diff
@@ -13,4 +13,6 @@
 # limitations under the License.
 ---
 containerd_version: "1.2.4"
-containerd_sha256: 329d44850685f8b511779c00763df50dd24257b3a1c594aff57e77bcee1b1269
+containerd_sha256: "8a2dc3cd9b2226f9703072ab0c18caca9a965d7e3bbf0de20e333406be513e78"
+cri_tools_version: "1.13.0"
```

Comment:
I'm wondering if we should leave these unset here and set/override them through packer like we do with `kubernetes_version` and `kubernetes_cni_version`, that is something that I can do as a followup, though.

Reply:
Very possibly. Agree it should be done as follow up though.